plugins/machina_tracker.user.js

```diff
@@ -0,0 +1,263 @@
+// @name           IITC Plugin: Machina tracker
```

Comment:
```suggestion
// @name           Machina tracker
```
The "IITC Plugin:" prefix is added automatically at build time

Reply:
also, you can drop the `user` from the filename